app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>BudgetApp</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= stylesheet_link_tag "custom", "data-turbo-track": "reload" %>
+    <%= javascript_importmap_tags %>
+  </head>
+
+  <body>
+    <div class="w-100">
+      <%= yield %>
+      <p class="notice text-success"><%= notice %></p>
+      <p class="alert"><%= alert %></p>
+    </div>
+    <%if user_signed_in? %>
+      <div class="btn-back">
```

Comment:
- Hey, from the main design, this back arrow button should be at the top left corner **ON ANY OF THE PAGES** you've used it. please, kindly fix this to improve your design as much as you can. 


![image](https://github.com/zbahati/budgetApp/assets/59397159/8a9597b6-c9af-4a33-906a-1dc9aac6bba7)



*http://127.0.0.1:3000/users/1/entities/1*
![image](https://github.com/zbahati/budgetApp/assets/59397159/c5635442-ee82-4322-851a-2f28d465f387)


NOTE: all pages you've used it should be moved up at the top left corner behind the header as in the image below:

*original design*
![image](https://github.com/zbahati/budgetApp/assets/59397159/4242d9ae-6970-4dcb-8696-d3670982447d)